### DISCUSSION TITLE
Standard format for recipes?

### DISCUSSION THREAD

**Discussion by zeekaran** [karlomikus/bar-assistant#299] [General]
Much outside the scope of bar-assistant, but I'm wondering if anyone has talked about making a recipe standard format that is designed sort of like how .epub was made (though that's a file type), so they can easily be shared between different programs, with future proofing in mind.

On another note, I have a bunch of recipes saved in plain text/markdown formatting. I manually wrote each recipe from a book that didn't have an ebook. I would be happy to have the option to write a script to convert it to something that was easily uploaded to bar-assistant as well as other apps, if they allowed as such. I think Drinkable said they were working on a recipe standard but I couldn't find it through google so maybe I hallucinated that. Does BA even have a mass/bulk importer?

**Comment by karlomikus** [comment]
Bar Assistant has a format that uses for exporting and importing recipes, but it's made mostly for it's backend. You can actually see this data when you export bar recipes, share a single recipe or check out the [initial data repository](https://github.com/bar-assistant/data).

As far as I know, there are a few recipe formats, but they are mostly aimed at cooking recipes, like [cooklang](https://cooklang.org/).

**Reply by karlomikus** [reply]
Also, schema.org has a [Recipe schema ](https://schema.org/Recipe)that pretty much all recipe websites have adopted and this seems the most standard one so far.
